Adyen/Core/APIClient/Environment.swift

```diff
@@ -7,51 +7,50 @@
 import Foundation
 
 /// :nodoc:
-/// Struct that defines the environment to retrieve resources from.
-public struct Environment: APIEnvironment {
+public protocol AnyAPIEnvironment {
```

Comment:
What is the point of this `protocol`? 🤔

Reply:
Yes, I was thinking of simplifying it to just be a `URL`. That would make things a bit easier. 
The protocol is also used in `internal enum DemoServerEnvironment: String, AnyAPIEnvironment`.
But `static let demoServerEnvironment = DemoServerEnvironment.test` is not used anywhere. i'm not sure what is going on.

Reply:
It should be used in `Demo/Common/Networking/DefaultAPIClient.swift`